packages/xstate-fsm/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "../../tsconfig.base.json",
   "include": ["src/**/*"],
   "compilerOptions": {
+    "strictFunctionTypes": true,
```

Comment:
To avoid merge conflicts with the `next` branch, I'd like to leave this off - even though that in general this is a good change. On the `next` branch we'll work on enabling this setting globally soon.

Reply:
Okay. I've reverted this change, disabled the `@ts-expect-error` comment as you suggested, and added a comment to the test saying that it isn't actually working at the moment.